src/fonduer/parser/visual_linker.py

```diff
@@ -16,6 +16,9 @@
 from fonduer.parser.models import Sentence
 from fonduer.utils.utils_visual import Bbox
 
+# Define a type alias for readability
+WordId = Tuple[int, int]
```

Comment:
Can you add documentation about these two arguments?

Reply:
Thanks! Resolved by a2bcb86a94f2a00d7a9bbaad73f96973f1c4fd14.